### PR TITLE
Romuva

### PR DESCRIPTION
**Pull request by asteraedm**

### CODE REVIEW

**Review by Chewyshoot — APPROVED** [review]
looks good, cheers to the baltic boiz

**Review by Chewyshoot — APPROVED** [review]
looks good, cheers to the baltic boiz

File: common/scripted_effects/pf_scripted_effects.txt [hunk 1]
```diff
@@ -109,4 +109,186 @@ pf_clear_trn_tribe_flags = {
...
+		limit = {
+			has_church_aspect = auslavis_healing_aspect
+		}
+		remove_church_aspect = auslavis_healing_aspect
+	}
+	if = {
+		limit = {
+			has_church_aspect = auslavis_renewal_aspect
+		}
+		remove_church_aspect = bangputys_renewal_aspect
```
Comment (maxice8) [inline common/scripted_effects/pf_scripted_effects.txt:208]: This and the 2 below are using the wrong aspect

File: common/scripted_effects/pf_scripted_effects.txt [hunk 2]
```diff
@@ -109,4 +109,186 @@ pf_clear_trn_tribe_flags = {
...
+	}
+	if = {
+		limit = {
+			has_church_aspect = dalia_wealth_aspect
+		}
+		remove_church_aspect = dalia_wealth_aspect
+	}
+	if = {
+		limit = {
+			has_church_aspect = ausrine_rebirth_aspect
```
Comment (maxice8) [inline common/scripted_effects/pf_scripted_effects.txt:272]: This and the 3 below are using the wrong aspect

File: events/RomuvaEvents.txt [hunk 3]
```diff
@@ -0,0 +1,331 @@
+# Romuva events
```
Comment (maxice8) [inline events/RomuvaEvents.txt:1]: Please name the file `pf_romuva_events` and use the same as namespace

**Review by maxice8 — CHANGES_REQUESTED** [review]

**Review by Chewyshoot — APPROVED** [review]

**Review by Chewyshoot — APPROVED** [review]

**Review by maxice8 — APPROVED** [review]

**Comment by maxice8** [comment]
Looks good, just need to solve the conflicts

**Review by Chewyshoot — APPROVED** [review]

**Review by Chewyshoot — APPROVED** [review]

**Pull request merged**